instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/rum/RumUtils.kt

```diff
@@ -79,8 +82,75 @@ fun sendRandomRumEvent(
                     forge.anActionName(),
                     defaultTestAttributes(testMethodName)
                 )
-                sendRandomActionOutcomeEvent(forge)
```

Comment:
Is this related with the fact that we are now accepting any action without necessarily checking for the outcome ?

Reply:
yes, we don't need side-effect now